src/obsidian/components/markdown-render-child/tldraw-markdown-render-child.ts

```diff
@@ -189,7 +193,16 @@ export class TldrawMarkdownRenderChild extends MarkdownRenderChild {
                                 onSelect(_) {
                                     editor.setCurrentTool(BoundsSelectorTool.id)
                                 },
-                            }
+                            },
+                            ['lasso-select']: {
+                                id: 'lasso-select',
+                                icon: 'tool-select',
+                                label: 'Lasso Select',
+                                kbd: 'w',
+                                onSelect: () => {
+                                    editor.setCurrentTool('lasso-select')
+                                },
+                            },
```

Comment:
Utilize the static `id` defined in the `LassoSelectTool` class
```suggestion
                            },
                            [LassoSelectTool.id]: {
                                id: LassoSelectTool.id,
                                icon: 'tool-select',
                                label: 'Lasso Select',
                                kbd: 'w',
                                onSelect: () => {
                                    editor.setCurrentTool(LassoSelectTool.id)
                                },
                            },
```